JS-112_javascript-arrays/store-multiple-values-in-an-array/code-challenge/access-array-elements-by-index/script.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * JS-112: JavaScript Arrays - Store Multiple Values In An Array
+ * 
+ * Code challenge: Access Array Elements By Index
+ */
+const players = ['Toni', 'Anwar', 'Mali', 'Carlos', 'Cormac', 'Sariah'];
+// Task 1: Log the first name inside the array using the console.log() method.
+console.log(players[0]);
+// Task 2: Use console.log() to log the last element inside the players array.
+console.log(players[5]);
```

Comment:
Although this is correct for a static array, this may return an error on a dynamic array. It would be safe to use the code below to get the last item of the array.
```suggestion
console.log(players[players.length - 1]);
```